TypeTreeDumper/RegisterDumperPluginAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace TypeTreeDumper
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
```

Comment:
I forgot to mention this earlier, but this should probably be changed to `[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]`

Reply:
Done